### PR TITLE
Access-control - EPeople admin page

### PR DESCRIPTION
**Pull request by MarieVerdonck**
This PR adds the admin access control pages for EPeople.

Functionalities:
- (Read) Initial page load gives you a pageable list of all EPeople. This list is searchable by scope, the default scope is byMetadata; byEmail is also selectable. Per EPerson ID, Name and Email is shown.
- (Create) EPerson form to create new EPerson
- (Delete) EPerson can be deleted
- (Update) EPerson form to edit their info (happens via patches to the backend)

This is dependent on backend PR: https://github.com/DSpace/DSpace/pull/2686. The Groups admin page will be coming in an upcoming PR.

### CODE REVIEW

File: src/app/+admin/admin-access-control/admin-access-control-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { EPeopleRegistryComponent } from './epeople-registry/epeople-registry.component';
+
+@NgModule({
+  imports: [
+    RouterModule.forChild([
+      { path: 'epeople', component: EPeopleRegistryComponent, data: { title: 'admin.access-control.epeople.title' } },
+    ])
+  ]
+})
+export class AdminAccessControlRoutingModule {
```
Comment (tdonohue) [inline src/app/+admin/admin-access-control/admin-access-control-routing.module.ts]: Please add TypeDocs to this new class

File: src/app/+admin/admin-access-control/admin-access-control.module.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    TranslateModule,
+    AdminAccessControlRoutingModule
+  ],
+  declarations: [
+    EPeopleRegistryComponent,
+    EPersonFormComponent
+  ],
+  entryComponents: []
+})
+export class AdminAccessControlModule {
```
Comment (tdonohue) [inline src/app/+admin/admin-access-control/admin-access-control.module.ts]: Please add TypeDocs to this new module

**Comment by benbosman** [comment]
Thanks @MarieVerdonck for the updates.

I've reviewed it again, and noticed from the changes:
* When trying to add a user whose email address already exists, the error is now displayed correctly. Thanks for that improvement
* When trying to alter a user, no data is sent to REST, no error is displayed, and the fields become blank. I did test this with the latest master REST

**Comment by MarieVerdonck** [comment]
- Fixed initial checkboxes not being checked correctly on first EPerson to edit
- Search by e-mail (exact) added
- Typedocs added
- Spec tests for service, registry, form and reducer
- Error notification if failing update/add eperson
- Specific error notification if failed due to email address in use
- Issue with currentUser email change giving failing message => Does get 403 from REST, despite the patch going through, see ^

**Pull request merged**